homeassistant/components/notify/pushbullet.py

```diff
@@ -24,27 +27,81 @@ def get_service(hass, config):
         return None
 
     try:
-        return PushBulletNotificationService(config[CONF_API_KEY])
-
+        pushbullet = PushBullet(config[CONF_API_KEY])
     except InvalidKeyError:
         _LOGGER.error(
             "Wrong API key supplied. "
             "Get it at https://www.pushbullet.com/account")
         return None
 
+    return PushBulletNotificationService(pushbullet)
+
 
 # pylint: disable=too-few-public-methods
 class PushBulletNotificationService(BaseNotificationService):
     """ Implements notification service for Pushbullet. """
 
-    def __init__(self, api_key):
-        from pushbullet import Pushbullet
+    def __init__(self, pb):
+        self.pushbullet = pb
+        self.pbtargets = {}
+        self.refresh()
 
-        self.pushbullet = Pushbullet(api_key)
+    def refresh(self):
+        '''
+        Refresh devices, contacts, channels, etc
 
-    def send_message(self, message="", **kwargs):
-        """ Send a message to a user. """
+        pbtargets stores all targets available from this pushbullet instance
+        into a dict. These are PB objects!. It sacrifices a bit of memory
+        for faster processing at send_message
+        '''
+        self.pushbullet.refresh()
+        self.pbtargets = {
+            'devices':
+                {tgt.nickname: tgt for tgt in self.pushbullet.devices},
+            'contacts':
+                {tgt.email: tgt for tgt in self.pushbullet.contacts},
+            'channels':
+                {tgt.channel_tag: tgt for tgt in self.pushbullet.channels},
+        }
 
+    def send_message(self, message=None, **kwargs):
+        """
+        Send a message to a specified target.
+        If no target specified, a 'normal' push will be sent to all devices
+        linked to the PB account.
+        """
+        targets = kwargs.get(ATTR_TARGET)
+        # Disabeling title
         title = kwargs.get(ATTR_TITLE)
+        title = None
```

Comment:
^ in the initial PR comment ;)
Title is set to None hard right now. Please feed back on this. The notify component sets the title to home assistent if not specified. This causes the message to be suppressed (in favor of the title) on mobile phones, etc when viewing the alert.

meaning, i have to unlock phone and open PushBullet app to see the message content.